Codeinwp/neve
# Range Components Fixes & Move to Hooks

### Summary
Moved Range and Responsive Range components to hook-based components.
Fixes issue where you can't type in the range input.
All controls have a min property of `0` unless the min property is less than 0.
You still can't delete the contents of the input and insert something else as core behavior is to reset it to the default value.
This behavior can be seen in the column block width range setting.


### Will affect visual aspect of the product

NO

### Test instructions


- You should be able to select the value and write something else in place.
- Min value should be hard coded to `0`, to allow writing values smaller than arbitrary min values we were previously using.
- This affects both Range and Responsive Range components.


Closes #1904.
